src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketTaskExtensions.cs

```diff
@@ -9,45 +10,64 @@ namespace System.Net.Sockets
 {
     public static class SocketTaskExtensions
     {
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Presumably all of our tests that were exercising the extension methods are now exercising the instance methods.

I suggest we add at least minimal "make sure the method is callable"-level tests for the extension then to, for example, validate we didn't inadvertently create any recursive-calls-leading-to-stack-overflow conditions with the extensions.

Reply:
Added